docs/guide/recommendation.md

```diff
@@ -230,21 +252,297 @@ You can also bypass named arguments using standard value passing as arguments.
+
+`Accept    application/vnd.ez.api.Content+json`
+
+Additionally you should check if the `contenttypes` interface is working as well by calling the following URI:
+
+`GET http://{endpoint}/api/ezp/v2/ez_recommendation/v1/contenttypes/38?page=1&page_size=10`
+
+`Accept    application/vnd.ez.api.Content+json`
+
+Both interfaces are supposed to provide content data in .json format.
+The difference is only the size of the content array in the `contentList` object.
+For the `content` interface one Content item is returned, for the `contenttypes` interface many are returned.
+
+``` json
+{
+    "contentList": {
+        "_media-type": "application/vnd.ez.api.contentList+json",
+        "content": [
+            {
+                "_media-type": "application/vnd.ez.api.content+json",
+                "contentId": 72,
+                "contentTypeId": 38,
+                "identifier": "place",
+                "language": "eng-GB",
+                "publishedDate": "2015-09-17T13:23:10+00:00",
+                "author": "Sandip Patel",
+                "uri": "/Places-Tastes/Places/Kochin-India",
+                "categoryPath": "/1/2/95/71/73/",
+                "mainLocation": {
+                    "_media-type": "application/vnd.ez.api.mainLocation+json",
+                    "_href": "/api/ezp/v2/content/locations/1/2/95/71/73/"
+                },
+                "locations": {
+                    "_media-type": "application/vnd.ez.api.locations+json",
+                    "_href": "/api/ezp/v2/content/objects/72/locations"
+                },
+                "name": "Kochin, India",
+                "intro": "<![CDATA[<section xmlns=\"http://ez.no/namespaces/ezpublish5/xhtml5\"><p>We got the major port city on the south west coast of India.</p></section>\n]]>",
+                "description": "<![CDATA[<section xmlns=\"http://ez.no/namespaces/ezpublish5/xhtml5\"><p><strong>Kochi </strong>(formerly Cochin) ... </p></section>\n]]>",
+                "image": "/var/site/storage/images/places-tastes/places/kochin-india/282-5-eng-GB/Kochin-India.jpg",
+                "caption": "<![CDATA[<section xmlns=\"http://ez.no/namespaces/ezpublish5/xhtml5\"><p>Chinese fishing nets ... </p></section>\n]]>",
+                "location": "kochin, india",
+                "authors_position": "Senior Editor",
+                "tags": "India, Kochin",
+                "rating": "0",
+                "publication_date": "1442500260",
+                "metas": ""
+            }
+        ],
+        ....
+    }
+}
+```
+
+## Running a full content export
+
+After defining what Content Types should be recommended and tracked you can start the full export with the following command:
+
+``` bash
+php ezpublish/console ezreco:runexport --contentTypeIdList=16 --webHook=https://admin.yoochoose.net/api/<your_customer_id>/items --hidden=1 --mandatorId=<your_customer_id> --host=<your_ezplatform_host_with_scheme>
```

Comment:
Could also be: (Symfony 2.8) `php app/console` / (Symfony >=3.x) `php bin/console`
(three other occurrences in this PR)